sdk-setup/src/mb2

```diff
@@ -4022,10 +4022,31 @@ fix_version_tc() {
     (
         git() (
             echo "stub invoked: git" >&2
-            if [[ $1 == stash && $2 == create ]]; then
+
+            options=()
+            while [[ $# -gt 0 ]]; do
+                case $1 in
+                    -c)
```

Comment:
What's this? Where would the -c come from?

Reply:
Check the commit referred from the commit message.

Reply:
You will also find why I didn't hit this before :)

Reply:
Right. I was clearly too tired on Friday as I didn't see it...